iree/samples/rt/vulkan/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ add_custom_command(
   COMMAND iree-translate
           --mlir-to-iree-module "${CMAKE_CURRENT_SOURCE_DIR}/simple_mul.mlir"
           -o simple_mul_bytecode_module.emod
-  DEPENDS iree::tools::iree_translate
+  DEPENDS iree_tools_iree_translate
```

Comment:
You could also resolve the
`# TODO(marbre): Add Vulkan dependencies`
TODO in this file by adding these lines:

https://github.com/google/iree/blob/14866cac626bbdadfd409e6ce7011056dd4c35c8/build_tools/third_party/dear_imgui/CMakeLists.txt#L56-L61

and adding the `Vulkan::Vulkan` dep to vk_graphics_integration.

(If any more parts of the code statically link in Vulkan we can centralize that check in some way)

Reply:
Good catch! Will push this in a separate commit.